go.mod

```diff
@@ -10,13 +10,9 @@ require (
 	github.com/go-openapi/strfmt v0.19.11
 	github.com/go-openapi/swag v0.19.12
 	github.com/go-openapi/validate v0.20.0
-	github.com/go-swagger/go-swagger v0.23.0
```

Comment:
i think you need to keep this dependency so that the `go run` for the `go:generate` instructions work.

the usual advice on how to do this is by creating a package called `tools` that has this:

```go
package tools

import (
  _ "github.com/go-swagger/go-swagger"
)
```

Reply:
if `go run pkg@version` is used then the do module is compiled and run without it being installed or affecting the current module. That is what I did. If you prefer I can change that back to what you are suggesting.

Reply:
> if `go run pkg@version` is used then the go module is compiled and run without it being installed or affecting the current module. That is what I did. If you prefer I can change that back to what you are suggesting.

ah good point. i'd prefer having it in `go.mod` so renovate can update it though

Reply:
Done